src/clarity/forms/_forms.clarity.scss

```diff
@@ -434,6 +433,23 @@ $clr-custom-checkbox-radio-top: ($clr_baselineRem_1 - $clr-checkbox-radio-height
             display: inline-block;
         }
 
+        input[type="checkbox"]:indeterminate + label::before {
```

Comment:
Yes, this code will work outside of clarity-angular **but,** the only way to set a checkbox to indeterminate state is with javascript. One cannot do 
`<input type="checkbox" indeterminate>`
like you can for checked. 
When it was brought to my attention this was referenced:
[Checkbox Visual States](https://css-tricks.com/indeterminate-checkboxes/)
[Pseudoclass Spec / Example](https://developer.mozilla.org/en-US/docs/Web/CSS/:indeterminate)
I'll add an example to the website. I need to wait until this is release though correct?